src/pages/Profile/Queue/Queues/Queues.jsx

```diff
@@ -55,7 +55,7 @@ export default function Queues({ queue, handleWrap, setSettings }) {
                             </RowContainer>
                             <RowContainer style={{ alignSelf: "flex-end", marginBottom: "15px" }}>
                                 <img src={group} /> 
-                                <ProfileTitleText style={{ marginLeft: "7px" }}> {/*queue.units.length*/} 0 </ProfileTitleText>
+                                <ProfileTitleText style={{ marginLeft: "7px" }}> { queue.units.filter( item => item.active === true).length } </ProfileTitleText>
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/OrangeMGE-ITP/SmartQ-ITP-Front/pullRequest?prid=8748018)
```suggestion
                                <ProfileTitleText style={{ marginLeft: "7px" }}> { queue.units.filter( (item) => item.active === true).length } </ProfileTitleText>
```